twill-yarn/src/main/java/org/apache/twill/internal/appmaster/RunningContainers.java

```diff
@@ -475,6 +496,11 @@ private void sendMessage(final String runnableName, final Message message,
     Futures.addCallback(controller.sendMessage(message), new FutureCallback<Message>() {
       @Override
       public void onSuccess(Message result) {
+        //update log level information
```

Comment:
onSuccess here really means that the message has been sent to the runnable successfully, right? But it does not reflect the actual state of the runnable itself? It that on purpose?